src/commonMain/kotlin/com/hedvig/authlib/network/LoginStatusResponse.kt

```diff
@@ -10,8 +10,8 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class LoginStatusResponse(
     val status: LoginStatus,
-    val statusText: String?,
-    val authorizationCode: String?
+    val statusText: String,
```

Comment:
Do we know that BankID sends an always non-null "statusText" in their response?
If not, we can always keep it nullable, but still forward it to the clients as a nullable thing.

Reply:
This is specified in auth service, and we default the status text to "Pending". So no dependency on BankID!